src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/UploadToolbarAction.js

```diff
@@ -131,20 +227,20 @@ export default class UploadToolbarAction extends AbstractListToolbarAction {
     }
 
     @computed get url(): string {
-        const {routeName} = this.options;
+        const {route_name: routeName} = this.options;
 
         if (typeof routeName !== 'string') {
-            throw new Error('The "routeName" option must be a string!');
+            throw new Error('The "route_name" option must be a string!');
         }
 
         return symfonyRouting.generate(routeName, this.requestParameters);
     }
 
-    @computed get errorCodeMapping(): $ReadOnly<Object> {
-        const {errorCodeMapping = {}} = this.options;
+    @computed get errorCodeMessages(): $ReadOnly<Object> {
```

Comment:
this makes the naming consistent with a similar option introduced in https://github.com/sulu/sulu/pull/5799